pkgs/by-name/bu/bundletool/package.nix

```
@@ -1,8 +1,17 @@
-{ lib, stdenvNoCC, fetchurl, makeBinaryWrapper, jre_headless }:
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  makeBinaryWrapper,
+  jre_headless,
+}:
 
-stdenvNoCC.mkDerivation rec {
-  pname = "bundletool";
+let
   version = "1.17.1";
+in
+stdenvNoCC.mkDerivation {
```

```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```
We could use `finalAttrs` pattern here.
With fixed point, other attributes are updated recursively when you override the package.

```nix
nix-repl> version = 1
nix-repl> ((stdenv.mkDerivation { pname = "test"; inherit version; passthru.version = version; }).overrideAttrs (_: { version = 2; })).version
1
nix-repl> ((stdenv.mkDerivation (finalAttrs: { pname = "test"; version = 1; passthru.version = finalAttrs.version; })).overrideAttrs (_: { version = 2; })).version   
2
```